src/qml/FeatureForm.qml

```diff
@@ -242,7 +242,8 @@ Page {
           top: fieldLabel.bottom
         }
 
-        text: !ConstraintHardValid ? qsTr( "Error: " ) + ConstraintDescription : !ConstraintSoftValid ? qsTr( "Warning: " ) + ConstraintDescription : ''
+        //text: !ConstraintHardValid ? qsTr( "Error: " ) + ConstraintDescription : !ConstraintSoftValid ? qsTr( "Warning: " ) + ConstraintDescription : ''
```

Comment:
Can you kill this line? It's NvPractice to comment, OgPractice is to trust the git log ;)

Reply:
Oh, I just forgot.